envs/r-dataethics.yaml

```diff
@@ -0,0 +1,19 @@
+'@context': 'https://stenci.la/schema/v1/'
+type: Environment
+name: r-dataethics
+description: >
+  An environment for the Data Ethics course run at the University of Canterbury in March-June 2019. The course is run
```

Comment:
Completely fine for me. Only small thing:
"Data Ethics course" should be "Data471 The Trustworthy Data Scientist"
(as it may conflict with other courses here)

Reply:
Updating